src/bin/gpfdist/regress/input/exttab1.source

```diff
@@ -650,6 +650,84 @@ SELECT * FROM readable_external_table_with_user_type_and_default_value;
 select * from exttab1_gpfdist_stop;
 -- end_ignore
 
+--
+-- Test default transformation
+--
+
+-- The server runs with default transformation of catfile_double which doubles
+-- file content. First server is for input flag, second is for output.
+CREATE EXTERNAL WEB TABLE gpfdist_transform_start1 (x text)
+execute E'((@bindir@/gpfdist -p 7070 -d @abs_srcdir@/data/ -c @abs_srcdir@/data/catfile.yml -I catfile_double </dev/null >/dev/null 2>&1 &); for i in `seq 1 30`; do curl 127.0.0.1:7070 >/dev/null 2>&1 && break; sleep 1; done; echo "starting...") '
+on SEGMENT 0
+FORMAT 'text' (delimiter '|');
+
+CREATE EXTERNAL WEB TABLE gpfdist_transform_start2 (x text)
+execute E'((@bindir@/gpfdist -p 7080 -d @abs_srcdir@/data/ -c @abs_srcdir@/data/catfile.yml -O catfile_double </dev/null >/dev/null 2>&1 &); for i in `seq 1 30`; do curl 127.0.0.1:7070 >/dev/null 2>&1 && break; sleep 1; done; echo "starting...") '
```

Comment:
Is it correct here to access port 7070 in `do curl 127.0.0.1:7070`, while gpfdist was started on 7080? Typo?

Reply:
Yes, it's a copied typo. Fixed.